### DISCUSSION TITLE
No document is generated after project build

### DISCUSSION THREAD

**Discussion by ankushmadankar** [Doraku/DefaultDocumentation#152] [General]
I have created fresh project and install Default Document [nuget ](https://www.nuget.org/packages/DefaultDocumentation) latest package (Version="0.8.2"). Updated the csproj file with following PropertyGroup

```
<PropertyGroup>
	..
	<GenerateDocumentationFile>true</GenerateDocumentationFile>
	<DefaultDocumentationFolder>Documentation\Api</DefaultDocumentationFolder>
</PropertyGroup>

```
then I build soultion inside Visual Studio (Version 17.8.3) and no document is created. I am expecting .md file at location `BuildLocation\Documentation\Api`. Anything I am missing?

Attached sample project [DefaultDocumentationTest.zip](https://github.com/Doraku/DefaultDocumentation/files/13776153/DefaultDocumentationTest.zip)

**Comment by ankushmadankar** [comment]
I suspect something broken with version 0.8.2. When I tried with 0.8.1 it worked fine.

**Comment by Doraku** [comment]
pretty late answer but the DefaultDocumentationFolder path is relative to the current directory, cleaning up old issues/discussion to keep relevant stuff, feel free to reopen if needed.
